chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/reviewer/SvapReviewer.java

```diff
@@ -45,14 +49,16 @@ public void review(CertifiedProductSearchDetails existingListing, CertifiedProdu
 
         if (!isListing2015Edition(existingListing)) {
             updatedListing.getCertificationResults().stream()
-                    .filter(cr -> cr.getSvaps() != null && cr.getSvaps().size() > 0)
-                    .forEach(cr -> updatedListing.getErrorMessages().add(
-                            errorMessageUtil.getMessage("listing.criteria.svap.invalidEdition",
-                                    cr.getNumber(), getListingEdition(existingListing))));
+            .filter(cr -> cr.getSvaps() != null && cr.getSvaps().size() > 0)
+            .forEach(cr -> updatedListing.getErrorMessages().add(
+                    errorMessageUtil.getMessage("listing.criteria.svap.invalidEdition",
+                            cr.getNumber(), getListingEdition(existingListing))));
         } else {
+            validateSvapNoticeUrl(updatedListing);
```

Comment:
Do we need a corresponding "pending" validator since the svap url can be set/edited during the confirm workflow?